docs/guidelines/accessibility.md

```diff
@@ -1,3 +1,659 @@
+## Accessibility
+
+## Web accessibility and Mass.gov
+
+Mass.gov is the website owned by the Commonwealth of Massachusetts.  As a government website, it is required to conform to the [Web Content Accessibility Guidelines (WCAG) 2.0 and 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/) and the [Americans with Disabilities Act (ADA)](https://www.ada.gov/).
+
+Mayflower is a design system for the Mass.gov website. It is designed and developed to meet such web acccessibility guideline.  If you follow this system's samples, your website would be accessible by broad range of users.
+
+As technology evolves, some methods to meet the guideline could get outdated. We will update outdated ones though our periodical accessibility testing.
+
+We value web accessibility, and consider it is important to promote it to anybody who own and work on websites. Here, you'll find resources for web accessibility from basic concepts to technical approaches.
+
+[TO DO:  ADD SOME QUICK LINKS TO RESOURCES.]
+
+
+## What is web accessibility?
+
+Web accessibility means **making your website available to as many people as possible**.
```

Comment:
maybe mention --> including search engines ?

Reply:
Since SEO improvement from accessibility is sub benefit, I think it's better to focus on accessibility itself here as its definition. The SEO benefit is mentioned in the Benefits of having an accessible website.